internal/cmd/git-combine/git-combine.go

```diff
@@ -81,7 +82,7 @@ func Combine(path string, opt Options) error {
 		}
 	}
 
-	logger.Println("Determining the tree hashes of subdirectories...")
+	logger.Info("Determining the tree hashes of subdirectories...")
```

Comment:
```suggestion
	logger.Debug("Determining the tree hashes of subdirectories...")
```

I think those comments are useful when you're trying to understand why something went wrong. We could get most of them to log at the Debug level and keep one as `Info` to still be able to see that `git-combine` did its job.

Reply:
We do have it deployed internally, mostly for testing purposes https://sourcegraph.sourcegraph.com/github.com/sourcegraph/deploy-sourcegraph-scaletesting@6ae4001fa89268732801ae6307feb0a9884a5a4e/-/blob/helm/sourcegraph/git-combine/git-combine.Service.yaml?L6:76-6:87 

Given how low effort this is, I'd rather have it log for that one time where we'll need it?